bindings/python/mata.pxd

```diff
@@ -93,8 +93,8 @@ cdef extern from "mata/nfa.hh" namespace "Mata::Nfa":
     ctypedef umap[State, State] StateToStateMap
     ctypedef umap[Symbol, string] SymbolToStringMap
     ctypedef umap[string, string] StringDict
-    ctypedef COrdVector[CTransSymbolStates] TransitionList
-    ctypedef vector[TransitionList] TransitionRelation
+    ctypedef COrdVector[CTransSymbolStates] MoveList
+    ctypedef vector[MoveList] TransitionRelation
```

Comment:
Why is this renamed?

Reply:
The original name was missleading, it was not a container of Transitions but of that things which now are renamed to Move. 
I am not sure that this name is the best option, vector is not a list, and in fact I am not sure whether  we need this type name at all.

Reply:
Hm since this is the automaton transtition relation, it should have a name.